ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.12.yaml

```diff
@@ -87,6 +87,7 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-az0
     env:
+      OPENSTACK_TEST_SKIPS: Amphora
```

Comment:
Amphora is not the correct keyword IMO, on the disconnected envs these tests will fail with OVN too [1]. Now using "OVN" as a keyword is probably too broad. I'd consider using this instead [2].

[1] https://github.com/openshift/openstack-test/blob/4b44bac5b9c49948a640eb783ed1eedc2384852d/test/extended/openstack/loadbalancers.go#L81
[2] https://github.com/openshift/openstack-test/blob/4b44bac5b9c49948a640eb783ed1eedc2384852d/test/extended/openstack/loadbalancers.go#L37

Reply:
`openstack][lb]` ?

Reply:
That would work for me.